python/QC3_Plotter.py

```diff
@@ -0,0 +1,310 @@
+
+import pandas as pd
+import xlrd
+from array import array
+import numpy as np
+import ROOT as r
+
+import argparse
+from optparse import OptionParser
+
+parser = argparse.ArgumentParser()
+parser.add_argument('filename', nargs='+')
+args = parser.parse_args()
+
+par1_Short= array( 'f' )
+par1_Long= array( 'f' )
+Err_par1_Short= array( 'f' )
+Err_par1_Long= array( 'f' )
+errX_Short=array('f')
+errX_Long=array('f')
+Detector_Short=array('f')
+Detector_Long=array('f')
+
+for f in args.filename:
```

Comment:
I think if you address:

https://github.com/bdorney/CMS_GEM_Analysis_Framework/issues/13

This will solve this problem and allow for enhanced functionality.